### PR TITLE
Feat auth context

### PR DESCRIPTION
**Pull request by JowelTisso**
* This PR is for auth context implementation :
* axios helper function also added
* token helper function also added
* minor changes in other files

### CODE REVIEW

**Review by pratik-kumar-sahu — APPROVED** [review]
- Its good that you are using localStorage for tokens.
- Else looks good to be merged.

File: src/src/utils/Constants.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Auth
+export const LOG_IN = "LOG_IN";
```
Comment (shraddha-amber) [inline src/src/utils/Constants.js:2]: suggestion: you can create an auth object and add these constants in that object

**Review by shraddha-amber — APPROVED** [review]
Simple and Modular code
Really liked the axiosHelper you created!

Looks great!

File: src/src/utils/Constants.js (hunk 1 above)
Comment (JowelTisso) [reply src/src/utils/Constants.js:2]: Ok will try that

**Pull request merged**